packages/react-components/src/components/VideoGallery/useVideoStreamLifecycleMaintainer.ts

```diff
@@ -49,17 +50,44 @@ const useVideoStreamLifecycleMaintainer = (
     onDisposeStreamView,
     renderElementExists,
     scalingMode,
-    streamId
+    streamId,
+    /* @conditional-compile-remove(media-access) */
+    isVideoPermitted
   } = props;
 
   const [videoStreamViewResult, setVideoStreamViewResult] = useState<CreateVideoStreamViewResult | undefined>();
 
+  const createStreamViewTrampoline = useCallback(
+    (
+      isStreamAvailable?: boolean,
+      renderElementExists?: boolean,
+      /* @conditional-compile-remove(media-access) */ isVideoPermitted?: boolean
+    ) => {
+      /* @conditional-compile-remove(media-access) */
+      if (isStreamAvailable && !renderElementExists && isVideoPermitted) {
```

Comment:
this check is going to cause bugs with the ability to create a stream if isVideoPermitted is undefined. This needs to be changed to all for the stream to be made if its undefined as the only time this typing should be blocking if its `false`

Reply:
Fixed, Thanks.